src/renderer/screens/dashboard/index.js

```diff
@@ -2,7 +2,7 @@
 import React, { useCallback, useMemo } from "react";
 import Box from "~/renderer/components/Box";
 import { accountsSelector } from "~/renderer/reducers/accounts";
-import BalanceSummary from "~/renderer/components/BalanceSummary";
```

Comment:
can you explain the shift to a new component and why BalanceSummary is not replaced by it? (twice code to maintain?)

Reply:
Currently this component is duplicated there and there (account has its own for instance). I think it need to be redesigned. I did not take the time to include it in this PR.

Reply:
@IAmMorrow can you add a reminder in our spreadsheet?